### PR TITLE
examples: remove PinMapClient class from _helpers.py

### PR DESCRIPTION
**Pull request by Jayaseelan-James**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?
This removes the PinMapClient class which is unused in the `_helpers.py`.

### Why should this Pull Request be merged?
With the "Update pin map" step, updating the pin map id to the Pin map service is now abstracted from the user. Hence, the PinMapClient class is not required in the `_helpers.py`.

### What testing has been done?
Ran mypy locally.

### CODE REVIEW

**Review by vigkre — APPROVED** [review]

File: examples/game_of_life/_helpers.py [hunk 1]
```diff
@@ -82,21 +48,10 @@ def get_active_pin_map_id(self) -> str:
         Returns:
             The resource id of the pin map that is registered to the pin map service.
         """
-        return self._sequence_context.Engine.TemporaryGlobals.GetValString(
```
Comment (Jayaseelan-James) [inline examples/game_of_life/_helpers.py:85]: FYI, the `Execution.RunTimeVariables` update and `set_active_pin_map` API removal were missed for the non-instruments in the earlier PRs. Hence, I'm accommodating those changes as well in this PR.

File: examples/game_of_life/_helpers.py
Comment (bkeryan) [inline examples/game_of_life/_helpers.py:1]: All copies of this file should be identical.

Please update the copies of this file in ni_measurementlink_generator. They are out of date.

Please update the copy of this file in nivisa_dmm_measurement to match the others.

```
PS D:\dev\measurementlink-python> cmd /c dir /s _helpers.py
 Volume in drive D is Images
 Volume Serial Number is 5469-81A8

 Directory of D:\dev\measurementlink-python\examples\game_of_life

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nidaqmx_analog_input

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nidcpower_source_dc_voltage

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nidigital_spi

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nidmm_measurement

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nifgen_standard_function

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\niscope_acquire_waveform

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\niswitch_control_relays

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\nivisa_dmm_measurement

12/12/2023  08:25 AM             3,589 _helpers.py
               1 File(s)          3,589 bytes

 Directory of D:\dev\measurementlink-python\examples\output_voltage_measurement

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\examples\sample_measurement

12/12/2023  08:25 AM             3,512 _helpers.py
               1 File(s)          3,512 bytes

 Directory of D:\dev\measurementlink-python\ni_measurementlink_generator\tests\test_assets\example_renders\measurement

11/14/2023  09:38 AM             5,328 _helpers.py
               1 File(s)          5,328 bytes

 Directory of D:\dev\measurementlink-python\ni_measurementlink_generator\tests\test_assets\example_renders\measurement_with_annotations

11/14/2023  09:38 AM             5,328 _helpers.py
               1 File(s)          5,328 bytes
```

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: examples/game_of_life/_helpers.py
Comment (bkeryan) [reply examples/game_of_life/_helpers.py:1]: Ok, ni_measurementlink_generator is covered by #551

Comment (bkeryan) [reply examples/game_of_life/_helpers.py:1]: The nivisa_dmm_measurement inconsistency is from running black manually as opposed to via ni-python-styleguide. I'll update it.
```
PS D:\dev\measurementlink-python> copy .\examples\nidcpower_source_dc_voltage\_helpers.py .\examples\nivisa_dmm_measurement\
PS D:\dev\measurementlink-python> git diff
diff --git a/examples/nivisa_dmm_measurement/_helpers.py b/examples/nivisa_dmm_measurement/_helpers.py
index 9f7056a6..21ef8057 100644
--- a/examples/nivisa_dmm_measurement/_helpers.py
+++ b/examples/nivisa_dmm_measurement/_helpers.py
@@ -65,13 +65,7 @@ class TestStandSupport(object):
         """
         if pathlib.Path(file_path).is_absolute():
             return file_path
-        (
-            _,
-            absolute_path,
-            _,
-            _,
-            user_canceled,
-        ) = self._sequence_context.Engine.FindFileEx(
+        (_, absolute_path, _, _, user_canceled) = self._sequence_context.Engine.FindFileEx(
             fileToFind=file_path,
             absolutePath=None,
             srchDirType=None,
```

**Review by bkeryan — APPROVED** [review]

File: examples/nivisa_dmm_measurement/_helpers.py [hunk 2]
```diff
@@ -100,13 +65,7 @@ def resolve_file_path(self, file_path: str) -> str:
...
             return file_path
-        (
-            _,
-            absolute_path,
-            _,
-            _,
-            user_canceled,
-        ) = self._sequence_context.Engine.FindFileEx(
+        (_, absolute_path, _, _, user_canceled) = self._sequence_context.Engine.FindFileEx(
             fileToFind=file_path,
```
Comment (keer1111) [inline examples/nivisa_dmm_measurement/_helpers.py:69]: is this indentation right?

**Review by keer1111 — APPROVED** [review]

File: examples/nivisa_dmm_measurement/_helpers.py (hunk 2 above)
Comment (bkeryan) [reply examples/nivisa_dmm_measurement/_helpers.py:69]: Yes, this is how black indents the code with `line-length = 100`. The other way is how black indents the code with the default `line-length`.

**Pull request merged**